src/search/open_lists/alternation_open_list.h

```diff
@@ -2,13 +2,13 @@
 #define OPEN_LISTS_ALTERNATION_OPEN_LIST_H
 
 #include "../open_list_factory.h"
-#include "../option_parser_util.h"
+#include "../plugins/options.h"
```

Comment:
Why do we include options.h and not plugin.h? (Same for all option parsers I believe, as well as lazy search and search common.)

Reply:
The code in this file doesn't need to know about plugins. If you would create an `Options` object manually, you could create an `AlternationOpenListFactory` from it. We only have to include `plugin.h` when we define a feature and a plugin for it.